src/SPIRV/Silk.NET.SPIRV.Reflect/Enums/AnonymousEnumSpirvReflectL281C1.gen.cs

```diff
@@ -10,8 +10,8 @@
 namespace Silk.NET.SPIRV.Reflect
 {
     [Flags]
-    [NativeName("Name", "__AnonymousEnum_spirv_reflect_L275_C1")]
-    public enum AnonymousEnumSpirvReflectL275C1 : int
+    [NativeName("Name", "__AnonymousEnum_spirv_reflect_L281_C1")]
+    public enum AnonymousEnumSpirvReflectL281C1 : int
```

Comment:
Is it possible to give this a slightly more sensible name? No idea what, but something that doesn't change every header update would be nice.